src/plugins/intel_gna/src/gna_plugin_config.cpp

```diff
@@ -171,7 +169,7 @@ OPENVINO_SUPPRESS_DEPRECATED_END
             } else if (value == PluginConfigParams::NO) {
                 gnaFlags.compact_mode = false;
             } else {
-                log << "GNA compact mode should be true/false (YES/NO), but not " << value;
+                ov::intel_gna::log::error() << "GNA compact mode should be true/false (YES/NO), but not " << value;
```

Comment:
Not sure that we need a log before exception

Reply:
Good catch ) removed